lib/src/layer/tile_layer/tile_coordinates.dart

```diff
@@ -2,6 +2,8 @@ import 'dart:math';
 
 import 'package:meta/meta.dart';
 
+/// The tile coordinates are based on maths' [Point] class but extended with
+/// the zoom level.
```

Comment:
Consider a brief explanation of how tile coordinates work, or a link to the documentation site: https://docs.fleaflet.dev/getting-started/explanation#slippy-map-convention?

Reply:
Added more informations. Feel free to improve or enhance. Not sure how to describe best so I added the link to the docs as well.